src/views/TabView.svelte

```diff
@@ -21,9 +21,9 @@
 
 <div class="flex w-full justify-center bg-white border-b-3 border-gray-200">
   <div class="flex">
-    {#each get(Menus.getMenuStore()) as menu, id}
+    {#each get(Menus.getMenuStore()) as menu}
       <a
-        class="block cursor-pointer mx-4 min-w-8rem py-2 px-12 border-b-4px border-b-white outline-none focus-visible:ring-4 focus-visible:ring-offset-1 focus-visible:ring-ring"
+        class="block cursor-pointer mx-4 min-w-8rem pb-2 pt-1 mt-1 px-12 border-b-4px border-b-white outline-none focus-visible:ring-4 focus-visible:ring-offset-1 focus-visible:ring-ring z-5"
```

Comment:
I think we can get away with z-2. Less likely to interfere with anything else.

Reply:
Good point, forgot to roll back now that it is no longer overlaying nav bar